docs/api-testing/composer/assertion-components.md

```diff
@@ -43,7 +43,7 @@ Allows you to compare two payloads in terms of text, structure or values.
 A **warning** will not trigger alerts, such as email or text messages.
```

Comment:
@aliciakrebs There are missing fields:
- Strict: Yes/No (default is No) | optional
- Comment: String | Optional